src/dev/mod.ts

```diff
@@ -10,6 +10,7 @@ import {
 import { error } from "./error.ts";
 
 const MIN_DENO_VERSION = "1.25.0";
+const TEST_FILE_PATTERN = /[\.|_]test[\.|_][t|j]s(x)?$/;
```

Comment:
```suggestion
const TEST_FILE_PATTERN = /[\.|_]test\.[t|j]s(x)?$/;
```

`foo_test_tsx` should not match. Can you add some unit tests for this regexp, or an integration test that ensures test files are ignored?

Reply:
The regex is updated and there is now a test for `collect`, which for now only tests that the test files are not included.